lib/stat_tracker.rb

```diff
@@ -172,12 +172,12 @@ def winningest_coach(season)
 
   def worst_coach(season) 
     season_comparer = season[0..3]
-    losing_games_in_season = @game_teams.find_all do |game_team| 
+    winning_games_in_season = @game_teams.find_all do |game_team| 
```

Comment:
Did not catch that the tests  was not corrected. The variable naming was also not up to date.